src/Psalm/Codebase.php

```diff
@@ -1406,6 +1406,7 @@ public function getCompletionItemsForClassishThing(string $type_string, string $
         $static_completion_items = [];
 
         $type = Type::parseString($type_string);
+        $snippets_supported = false;
```

Comment:
This should come from `clientCapabilities.textDocument.completion.completionItem.snippetSupport`.

Reply:
Thanks. I hope that anyone more familiar with Psalm will help figuring out how to get at that value, since I'm not quite sure...

Reply:
@matthijskooijman these are here https://github.com/vimeo/psalm/blob/b6a32825894a9b4c6d4638f59b7c9db3bf933fbc/src/Psalm/Internal/LanguageServer/LanguageServer.php#L187
You need to store the required capabilities at `TextDocument` and use it there if you want it configurable.

Reply:
Yeah, I found that one, but AFAICS the `LanguageServer\ClientCapabilities` object does not actually store the values I need, so maybe that library needs changes? See https://github.com/felixfbecker/php-language-server-protocol/blob/master/src/ClientCapabilities.php

Reply:
It seems so, yes. That library looks barely alive, no wonder something is missing. It is out of Psalm's scope though.

Reply:
I do see some activity a few months ago, so it doesn't seem completely dead. Maybe supplying a PR for this to php-language-server-protocol would work, I suspect that it would be fairly trivial code. However, this is way too far into yak-shaving-territory for me, so I won't be picking this up anytime soon...